app/services/analytics.rb

```diff
@@ -292,5 +292,6 @@ def browser_attributes
   VENDOR_OUTAGE = 'Vendor Outage'
   WEBAUTHN_DELETED = 'WebAuthn Deleted'
   WEBAUTHN_SETUP_VISIT = 'WebAuthn Setup Visited'
+  WEBAUTHN_AUTHENTICATION_VISIT = 'WebAuthn Authentication Visited'
```

Comment:
Minor: Can we alphabetize this?

```suggestion
  WEBAUTHN_AUTHENTICATION_VISIT = 'WebAuthn Authentication Visited'
  WEBAUTHN_DELETED = 'WebAuthn Deleted'
  WEBAUTHN_SETUP_VISIT = 'WebAuthn Setup Visited'
```